### ISSUE TITLE
Promotional Code with Free Shipping bugs when All In One Shipping-extension is installed

### ISSUE THREAD

**Issue by bos4711** [cubecart/v6#2098]
This issue is maybe related to #1791 (which is marked fixed). Quoting the last paragraph:

> selecting "Free Coupon Shipping" from the new shipping method refreshes the screen and always resets back to "Please select". You actually can't checkout with a free shipping coupon unless you enable the setting "Allow physical orders even if no shipping options are available"

Even with "Allow physical orders ..." checked, the refresh-loop clears the selected "Free Coupon Shipping" and you cannot check-out the order, if the "All in one shipping"-extension is installed. I tried to disable the extension for a short while to see if it still bugs out, but somehow I cannot disable it.

One workaround here would ofcourse be to - in the extension - add an option for free shipping (useful for local pick-up, for example). The problem with doing that is that since the shipping cost is zero (0), that option gets auto-selected, meaning that all orders will get free shipping.

TL;DR: Using a Promotional Code with "Free shipping" enabled does not work when the free extension "All in one shipping" is installed.
